### DISCUSSION TITLE
One small issue link to one pull request

### DISCUSSION THREAD

**Discussion by jongfeel** [wisdom08/dev-community#10] [General]
issue에 여러 개의 task 목록을 만드셨는데
하나의 task에 하나의 pull request를 만들어 보는 게 좋을 것 같습니다.

**Comment by wisdom08** [comment]
@jongfeel 
아 이슈의 크기가 너무 큰 게 문제군요. 
[이슈#4](https://github.com/wisdom08/dev-community/issues/4) 의 각 체크박스를 하나의 이슈로 만드는 게 적절한 크기였을까요?

**Reply by jongfeel** [reply]
네 하나의 pull request로 만드는 게 더 좋지 않았을까 하는 생각입니다

**Reply by wisdom08** [reply]
넵 다음 이슈는 크기를 적당하게 쪼개서 만들어보도록 하겠습니다.
